_includes/footer.html

```diff
@@ -34,5 +34,4 @@ <h3 class="fancy">
             </ul>
         </div>
     </section>
-    <script src="/chai.js" type="text/javascript"></script>
```

Comment:
Just noticed this. We should ideally keep chai loaded on every page so that users who visit the site can play around with it from the devtools. We could certainly use the cdn version, and drop the makefile line (as you mention in https://github.com/chaijs/chai-docs/pull/132#issuecomment-219617718).